.github/workflows/docker-image.yml

```diff
@@ -31,12 +31,22 @@ jobs:
       - name: install semver tool
         run: pip3 install semver
 
+      - name: strip v from tag
+        run: |
+          if [ ${{ github.event_name }} == 'release' ]
+          then
+            echo "Strip v from tag"
+            LAST_PART=$(echo $GITHUB_REF | cut -d / -f 3)
+            STRIPPED_VAR="${LAST_PART:1}"
```

Comment:
I think we need a validation that the tag actually starts with a v. if not, this part would turn `15.2.4` into `5.2.4` and the rest of the workflow would nof fail.